history-of-the-quakers-v4/original/25-book-8-chapter-2.adoc

```diff
@@ -584,6 +611,9 @@ read and approved in the Quarterly-Meeting for Ulster,
 held at Moyallon the 18th of 2nd Month,
 and in the National Half-year`'s Meeting in Dublin, 5th Month, 1792.
 
+[.embedded-content-document.testimony]
```

Comment:
I can't leave a comment on the line, but above here, let's insert an asterism before line 604, and then lets' make the line starting "A Testimony from Lisburn..." into a subheading blurb style as well 👍